### DISCUSSION TITLE
Question: Multivariate timeseries with feat_dynamic_real

### DISCUSSION THREAD

**Discussion by ghost** [awslabs/gluonts#1308] [Q&A]
I'm trying to forecast for a small set of multivariate timeseries. They have dynamic features associated with some of them. I want to alter those dynamic features during predictions so that I can simulate forecasts under different future conditions. 

I've tried `GPVAR`, and it doesn't seem to take any dynamic features right now. While adding those features directly as timeseries produced reasonable forecasts, it doesn't allow me to simulate forecasts under different future conditions. 

I also tried `DeepAR` which takes in dynamic features but didn't produce good forecasts. I suppose that's because it assumes univariate timeseries. 

Is there another model I can use that allows feat_dynamic_real. Or is there some relatively easy ways I can change the backend code to make this possible? I'm pretty new to this package, would appreciate someone points me to the right package or places.

**Comment by jaschau** [comment]
Hi @hweiTPR, I've been using the hack outlined here https://github.com/awslabs/gluon-ts/issues/652#issuecomment-591323497 in the past to make GPVAR work with dynamic features. Maybe you can give it a try.
